dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/ContentletDependencies.java

```diff
@@ -82,13 +98,51 @@ public IndexPolicy getIndexPolicyDependencies() {
         return permissions;
     }
 
+    public String getWorkflowPublishDate() { return workflowPublishDate; }
+
+    public String getWorkflowPublishTime() {
+        return workflowPublishTime;
+    }
+
+    public String getWorkflowExpireDate() {
+        return workflowExpireDate;
+    }
+
+    public String getWorkflowExpireTime() {
+        return workflowExpireTime;
+    }
+
+    public String getWorkflowNeverExpire() {
+        return workflowNeverExpire;
+    }
+
+    public String getWorkflowWhereToSend() {
+        return workflowWhereToSend;
+    }
+
+    public String getWorkflowFilterKey() {
+        return workflowFilterKey;
+    }
+
+    public String getWorkflowIWantTo() {
+        return workflowIWantTo;
+    }
+
     public static final class Builder {
 
         private User modUser;
         private ContentletRelationships relationships;
         private String workflowActionId;
         private String workflowActionComments;
         private String workflowAssignKey;
+        private String workflowPublishDate;
+        private String workflowPublishTime;
```

Comment:
Codacy found an issue: [Field workflowPublishTime has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5919676)